Felienne/hedy
# Refactor CSS for /stats accordion #1242

**Description**
Removed unnecessary CSS code.

**Fix for**

#1242 

**How to test**

- The user should be able to open and close the tabs of the accordion on the /stats page.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
